src/dbGetData.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace TheRealMVP;
+use PDO;
+
+
+/**Pulls info from the database
+ *
+ * @param $connection & $query
+ *
+ * @return string & integer
+ */
+class dbGetData
```

Comment:
Again, I think this function needs to be inside the TeamHydrator class? And delete the dbGetData class? 

Then, the TeamHydrator class with connect to the DB, get the info out, and hydrate the Team objects?

Reply:
But classes are only meant to do one thing, we thought? That's why we've kept them separate.